inc/networkinventory.class.php

```diff
@@ -367,7 +367,10 @@ public function run($jobstate)
         $current = $jobstate;
         $agent->getFromDB($current->fields['agents_id']);
 
-        $ip = $this->getDeviceIPOfTaskID($jobstate->fields['itemtype'], $jobstate->fields['items_id'], $jobstate->fields['plugin_glpiinventory_taskjobs_id']);
+        $taskjob = new PluginGlpiinventoryTaskjob();
+        $taskjob->getFromDB($jobstate->fields['plugin_glpiinventory_taskjobs_id']);
+
```

Comment:
```suggestion
$current = $jobstate;
```

Reply:
`$current` is used later we need to update it to make sure it contains the good `jobstate`